_posts/2017-02-19-simplified-database-migrations-in-go.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: "Simplified database migrations in go"
+ghavatar: 501424
+ghname: mastoj
+jobtitle: Platform Engineer
+tags: [go, golang]
+---
+
+## Introduction
+
+There are two main criteria that one should look for when choosing a database migration framework; it should be simple and it should always roll forward. When we began looking for a migration framework to use in one of our applications we couldn't find one that was simple in the sense that it could read from file as well as from pre-compiled assets, which we tend to use at Unacast, and only rolls forward. As a result of that we ended up implementing our own simple [migration framework](https://github.com/unacast/migrations) for go applications, which I will guide you through in this post.
```

Comment:
Maybe add Go in the "in one of our applications" section so it's clear that we'ew talking about Go here?